pkg/apis/pipeline/v1beta1/pipelinerun_validation.go

```diff
@@ -107,14 +110,79 @@ func (ps *PipelineRunSpec) Validate(ctx context.Context) (errs *apis.FieldError)
 			wsNames[ws.Name] = idx
 		}
 	}
-
 	for idx, trs := range ps.TaskRunSpecs {
 		errs = errs.Also(validateTaskRunSpec(ctx, trs).ViaIndex(idx).ViaField("taskRunSpecs"))
 	}
 
 	return errs
 }
 
+func (ps *PipelineRunSpec) validateInlineParameters(ctx context.Context) (errs *apis.FieldError) {
```

Comment:
could you please add some comments for this function describing what is being validated?
Also, can this be consolidated with existing logic for param validation?